satpy/modifiers/atmosphere.py

```diff
@@ -31,7 +31,41 @@
 
 
 class PSPRayleighReflectance(ModifierBase):
-    """Pyspectral-based rayleigh corrector for visible channels."""
+    """Pyspectral-based rayleigh corrector for visible channels.
+
+    It is possible to use ``lim_low``, ``lim_high``
+    and ``strength`` together to reduce rayleigh correction at high solar zenith angle
+    and make the image transition from rayleigh-corrected to partially/none rayleigh-corrected
+    at day/night edge, therefore producing a more natural look. This reduction starts at
+    solar zenith angle of ``lim_low``, and ends in ``lim_high``. It's linearly scaled
+    between these two angles. The ``strength`` controls the amount of the reduction. When
+    the solar zenith angle reaches ``lim_high``, the rayleigh correction will remain
+    ``(1 - strength)`` of its initial strength.
+
+    To use this function in a YAML configuration file:
+
+    .. code-block:: yaml
+
+      rayleigh_corrected_reduced:
+        modifier: !!python/name:satpy.modifiers.PSPRayleighReflectance
+        atmosphere: us-standard
+        aerosol_type: rayleigh_only
+        lim_low: 70
+        lim_high: 95
+        strength: 0.5
```

Comment:
I'm wondering whether these shall be renamed to make it more clear what their meaning is. Perhaps simply `reduce_lim_low`, `reduce_lim_high` and `reduce_strength`?